projects/packages/scheduled-updates/changelog/update-modify-plugin-endpoint

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: changed
+
+Pass is_scheduled_updats property in the filter to identify our requests
```

Comment:
The changelog entry will be read by developers and users that will not have the context of this PR. Would you mind taking another pass at this, adding more context, and making it valuable for folks who don't know what we're working on?

Reply:
I've modified the changelog and added more context, thanks!